### ISSUE TITLE
Throws warnings in playerctl

### ISSUE THREAD

**Issue by samhh** [jgierer12/pocket-casts-linux#1]
Hi there!

I use a [script](https://github.com/SamHH/dotfiles/blob/master/linux/home/scripts/music.sh) to echo the currently playing media in my bar and noticed that it doesn't work whenever this app is open. I figured out it's because of the warnings/errors being thrown when calling `playerctl status`:

```
(playerctl:15451): GLib-GIO-WARNING **: 16:36:27.412: Received property Rate with type y does not match expected type d in the expected interface

(playerctl:15451): GLib-GIO-WARNING **: 16:36:27.412: Received property Volume with type y does not match expected type d in the expected interface

(playerctl:15451): GLib-GIO-WARNING **: 16:36:27.412: Received property Position with type y does not match expected type x in the expected interface

(playerctl:15451): GLib-GIO-WARNING **: 16:36:27.413: Received property MinimumRate with type y does not match expected type d in the expected interface

(playerctl:15451): GLib-GIO-WARNING **: 16:36:27.413: Received property MaximumRate with type y does not match expected type d in the expected interface
Paused
```

This happens regardless of whether the app is playing or paused, and does not happen when the app is closed (i.e. cmus does not give me these errors).

Cheers!

**Comment by jgierer12** [comment]
Hi, thanks for reporting!

I'll need to investigate why those warnings are thrown, it might be an upstream issue with [emersion/mpris-service](https://github.com/emersion/mpris-service). It just got a major version bump and claims to fix many issues, so it may simply be a matter of upgrading that lib to 2.0.0.

**Comment by jgierer12** [comment]
Fixed in #2; Upgrading mpris-service to 2.0.0 got rid of the warnings.
Released in [`v1.1.0`](https://github.com/jgierer12/pocket-casts-linux/releases/tag/v1.1.0).

**Comment by samhh** [comment]
Works perfectly, thank you! :100:
